components/button/index.css

```diff
@@ -106,6 +104,258 @@ governing permissions and limitations under the License.
+    --spectrum-button-background-color-hover: var(--spectrum-accent-background-color-hover);
+    --spectrum-button-background-color-down: var(--spectrum-accent-background-color-down);
+    --spectrum-button-background-color-focus: var(--spectrum-accent-background-color-key-focus);
+
+    &.spectrum-Button--outline {
+      --spectrum-button-background-color-default: transparent;
+      --spectrum-button-background-color-hover: var(--spectrum-accent-color-200);
+      --spectrum-button-background-color-down: var(--spectrum-accent-color-300);
+      --spectrum-button-background-color-focus: var(--spectrum-accent-color-200);
+
+      --spectrum-button-border-color-default: var(--spectrum-accent-color-900);
+      --spectrum-button-border-color-hover: var(--spectrum-accent-color-1000);
+      --spectrum-button-border-color-down: var(--spectrum-accent-color-1100);
+      --spectrum-button-border-color-focus: var(--spectrum-accent-color-1000);
+
+      --spectrum-button-content-color-default: var(--spectrum-accent-content-color-default);
+      --spectrum-button-content-color-hover: var(--spectrum-accent-content-color-hover);
+      --spectrum-button-content-color-down: var(--spectrum-accent-content-color-down);
+      --spectrum-button-content-color-focus: var(--spectrum-accent-content-color-key-focus);
+    }
+  }
+
+  /* ---- Negative ---- */
+  &.spectrum-Button--negative {
+    --spectrum-button-background-color-default: var(--spectrum-negative-background-color-default);
+    --spectrum-button-background-color-hover: var(--spectrum-negative-background-color-hover);
+    --spectrum-button-background-color-down: var(--spectrum-negative-background-color-down);
+    --spectrum-button-background-color-focus: var(--spectrum-negative-background-color-key-focus);
+
+    &.spectrum-Button--outline {
+      --spectrum-button-background-color-default: transparent;
+      --spectrum-button-background-color-hover: var(--spectrum-negative-color-200);
+      --spectrum-button-background-color-down: var(--spectrum-negative-color-300);
+      --spectrum-button-background-color-focus: var(--spectrum-negative-color-200);
+
+      --spectrum-button-border-color-default: var(--spectrum-negative-color-900);
+      --spectrum-button-border-color-hover: var(--spectrum-negative-color-1000);
+      --spectrum-button-border-color-down: var(--spectrum-negative-color-1100);
+      --spectrum-button-border-color-focus: var(--spectrum-negative-color-1000);
+
+      --spectrum-button-content-color-default: var(--spectrum-negative-content-color-default);
+      --spectrum-button-content-color-hover: var(--spectrum-negative-content-color-hover);
+      --spectrum-button-content-color-down: var(--spectrum-negative-content-color-down);
+      --spectrum-button-content-color-focus: var(--spectrum-negative-content-color-key-focus);
+    }
+  }
+
+  /* ---- Primary ---- */
+  /* Also shows as the default when a variant is not set. */
+  &.spectrum-Button--primary,
+  & {
```

Comment:
Do we need to continue supporting the `.spectrum-Button--primary` class if these styles are the default on `.spectrum-Button`?

Reply:
You're right that it isn't necessary to have both, if one is the default. 

I wasn't sure about whether this change was the best course of action or which variant would be the default. Or how that might affect SWC. I was hesitant to remove the primary class without knowing. I'll start a Slack discussion about this to get some more opinions.

Previously, if you didn't have one of the variant classes on the button, it does not display with any of the designed colors. So basically one of the variant classes is required. It seems like it would be a good idea to have the default display nicely if there is no variant class.

Reply:
After our discussion today, I've made "accent" the default, as accent shows as the default on the guidelines. And I  have kept the class around that is used for the default, to avoid a drastic change here but also leaving the option to remove it in the future.